pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "Cython>=3.0.0", "numpy>=1.26.0"]
+requires = ["setuptools>=75.6", "wheel", "Cython>=3.0.8", "numpy>=1.26.3"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "matchmaker"
-version = "0.1.0"
+name = "pymatchmaker"
+version = "0.1.1"
```

Comment:
I've updated the version https://github.com/pymatchmaker/matchmaker/commit/7b3211e95b1fcc5b8378607cd2dd160b089374db